engine/runtime/src/test/resources/TestNonImportedOverloads/src/Util.enso

```diff
@@ -1,5 +1,6 @@
 import Standard.Base.Nothing
 
-type X a
+type X
+    Mk_X a
```

Comment:
```suggestion
    X_Data a
```

Reply:
These don't really matter now, these are internal tests in the interpreter, not even the stdlib tests. Will revisit them when statics appear.